### PR TITLE
Updates to AMY for DFO notebook

### PR DESCRIPTION
**Pull request by nicolejkeeney**
Among other things, creates a new, user friendly function for accessing AMY data. Modifies how data is accessed to allow greater flexibility with AMY-related analyses. Renames tmy.py to meteo_yr.py. **BEFORE MERGING TO MAIN, WE WILL NEED TO UPDATE THE EXISTING TMY NOTEBOOK TO RENAME THE FUNCTIONS**

### CODE REVIEW

**Review by vicford — CHANGES_REQUESTED** [review]
Looks like the data location map in the explore AMY panel is shifted to the right? 

Screen Shot 2022-12-16 at 9 36 00 AM

While also not necessary, I am wondering whether this might be a good time to also clean up the meteo_yr.py script to switch all remaining instances of "TMY" to "AMY", this will require quite a bit of checking to make sure nothing breaks, food for thought

**Comment by vicford** [comment]
I also have pushed a PR on the AMY notebook as requested :)

File: climakitae/data_loaders.py [hunk 1]
```diff
@@ -298,7 +298,7 @@ def _get_data_one_var(selections, location, cat):
             dset = dset.rio.clip(geometries=ds_region, crs=4326, drop=True)
 
         # Perform area averaging
-        if selections.area_average == True:
+        if selections.area_average == "Yes":
```
Comment (elehmer) [inline climakitae/data_loaders.py:301]: Ahh yeah this is different now...

**Review by elehmer — APPROVED** [review]
LGTM!

**Comment by elehmer** [comment]
> Ran through the entire notebook with this climakitae branch loaded in, everything now works as it should with updated language for the notebook in PR [#30](https://github.com/cal-adapt/cae-notebooks/pull/30).
> 
> Looks like the data location map in the explore AMY panel is shifted to the right?
> 
> Screen Shot 2022-12-16 at 9 36 00 AM
> 
> While also not necessary, I am wondering whether this might be a good time to also clean up the meteo_yr.py script to switch all remaining instances of "TMY" to "AMY", this will require quite a bit of checking to make sure nothing breaks, food for thought

Hmm, yeah I'm seeing the offset map as well. Annoying!!

**Review by vicford — APPROVED** [review]

**Pull request merged**